src/metabase/api/pulse.clj

```diff
@@ -106,18 +103,17 @@
     (assert (integer? card-id))
     (api/read-check Card card-id)))
 
-#_{:clj-kondo/ignore [:deprecated-var]}
-(api/defendpoint-schema POST "/"
+(api/defendpoint POST "/"
   "Create a new `Pulse`."
   [:as {{:keys [name cards channels skip_if_empty collection_id collection_position dashboard_id parameters]} :body}]
-  {name                su/NonBlankString
-   cards               (su/non-empty [pulse/CoercibleToCardRef])
-   channels            (su/non-empty [su/Map])
-   skip_if_empty       (s/maybe s/Bool)
-   collection_id       (s/maybe su/IntGreaterThanZero)
-   collection_position (s/maybe su/IntGreaterThanZero)
-   dashboard_id        (s/maybe su/IntGreaterThanZero)
-   parameters          [su/Map]}
+  {name                ms/NonBlankString
+   cards               [:+ pulse/CoercibleToCardRef]
+   channels            [:+ :map]
+   skip_if_empty       [:maybe :boolean]
+   collection_id       [:maybe ms/PositiveInt]
+   collection_position [:maybe ms/PositiveInt]
+   dashboard_id        [:maybe ms/PositiveInt]
+   parameters          [:maybe [:sequential :map]]}
```

Comment:
I think I had a failed test for this because there is a difference between `[su/Map]` and `[:sequential :map]`

for the `[su/Map]`, `nil` is a valid value. that's why I needed to add `[:maybe]`.

I think that was unexpected in most cases tho.